### PR TITLE
update dependencies in lockfile for 7.9.0

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

**Comment by jsvd** [comment]
jenkins test this please

**Comment by jsvd** [comment]
test failure is related to a change in a new release of the ES output, a workaround fix exists in https://github.com/elastic/logstash/pull/12113

File: Gemfile.jruby-2.5.lock.release [hunk 1]
```diff
@@ -461,12 +464,13 @@ GEM
       logstash-filter-json
       logstash-input-generator
       logstash-output-file
-    logstash-output-elastic_app_search (1.0.0)
+    logstash-output-elastic_app_search (1.1.0)
       logstash-codec-plain
       logstash-core-plugin-api (~> 2.0)
-    logstash-output-elasticsearch (10.5.1-java)
+    logstash-output-elasticsearch (10.6.0-java)
```
Comment (colinsurprenant) [inline Gemfile.jruby-2.5.lock.release:470]: Per discussion with @jsvd - until we push the fix which currently breaks the build with the lastest version of the ES output, let's keep the previous version for now to have a green build on 7.9. 
```suggestion
    logstash-output-elasticsearch (10.5.1-java)
```

**Comment by colinsurprenant** [comment]
LGTM with the exception of the inline comment about the ES output version revert for now until a fix is pushed as discussed with @jsvd.

File: Gemfile.jruby-2.5.lock.release [hunk 2]
```diff
@@ -461,12 +464,13 @@ GEM
...
       logstash-input-generator
       logstash-output-file
-    logstash-output-elastic_app_search (1.0.0)
+    logstash-output-elastic_app_search (1.1.0)
       logstash-codec-plain
       logstash-core-plugin-api (~> 2.0)
     logstash-output-elasticsearch (10.5.1-java)
       cabin (~> 0.6)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
+      logstash-mixin-ecs_compatibility_support (~> 1.0)
```
Comment (jsvd) [inline Gemfile.jruby-2.5.lock.release:473]:
```suggestion
```

File: Gemfile.jruby-2.5.lock.release [hunk 3]
```diff
@@ -448,6 +449,8 @@ GEM
       aws-sdk-v1 (>= 1.61.0)
       logstash-codec-plain
       logstash-core-plugin-api (>= 1.60, <= 2.99)
+    logstash-mixin-ecs_compatibility_support (1.0.0-java)
+      logstash-core (>= 5.0.0)
```
Comment (jsvd) [inline Gemfile.jruby-2.5.lock.release:453]:
```suggestion
```

**Comment by jsvd** [comment]
@colinsurprenant I committed two more changes, to remove the dependency that the new version introduced.

**Pull request merged**